### DISCUSSION TITLE
Analytics events in production not shown

### DISCUSSION THREAD

**Discussion by artkrv** [invertase/react-native-firebase#7453] [Q&A]
I have an application with analytics react-native-firebase/analytics installed and custom events setup.
I can see the events firing in DebugView, but in released app (android/ios) shows only the default events:
<img width="194" alt="image" src="https://github.com/invertase/react-native-firebase/assets/12414560/56011f77-5e44-428d-bd75-9a1c8da9fbc4">
Any suggestion acceptable to start digging.

Thanks.

**Comment by events-jonas-chrisw** [comment]
I have the same problem here - custom events aren't logging but default events are..

**Comment by ricardoolivaresventura** [comment]
I have the same problem

**Comment by dancherb** [comment]
Did you resolve this? For me, custom events also aren't logging when fired from my local environment. But standard events like "screen_view" are. @ricardoolivaresventura @events-jonas-chrisw

**Comment by artkrv** [comment]
I think I  marked events as conversions. Analytics conversions have been renamed key events.
